masterlist.yaml

```
@@ -13983,6 +13983,10 @@ plugins:
       - 'CoW Entry Requirements.esp'
       - 'NotSoFast-MageGuild.esp'
       - 'MCWT_TheAugursAdmonishment.esp'
+    msg:
+      - <<: *patchProvided
+        subs: ['College of Winterhold - Quest Expansion']
```

There should be spaces separating the square brackets & single quotes:

```suggestion
        subs: [ 'College of Winterhold - Quest Expansion' ]
```